roles/pigeon/templates/mainnet/config.yaml.j2

```diff
@@ -16,55 +16,55 @@ evm:
     chain-id: 1
     base-rpc-url: {{ vault.mainnet.paloma.eth_rpc }}
     keyring-pass-env-name: KEYRING_PASSWORD
-    signing-key: SIGNING_KEY
+    signing-key: ${SIGNING_KEY}
```

Comment:
do we need to add it in ${} ? KEYRING_PASSWORD works without it

Reply:
what we can do it use ansible lookup and substitute the variable at ansible runtime, a general env wont be needed in that case, we can just define it with the inventory